cmd/check-operator/patchcrd/patchcrd.go

```diff
@@ -0,0 +1,120 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package patchcrd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/DataDog/datadog-operator/pkg/plugin/common"
+)
+
+// Options provides information required to manage the patch function.
+type Options struct {
+	genericclioptions.IOStreams
+	common.Options
+	args           []string
+	crdName        string
+	storedVersions []string
+}
+
+// NewOptions provides an instance of Options with default values.
+func NewOptions(streams genericclioptions.IOStreams) *Options {
+	opts := &Options{
+		IOStreams: streams,
+	}
+
+	opts.SetConfigFlags()
+
+	return opts
+}
+
+// NewCmdPatch provides a cobra command wrapping Options.
+func NewCmdPatch(streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:          "patch-crd <crd-name>",
+		Short:        "patch the CRD status to update storedVersions list",
+		Example:      "./check-operator patch-crd datadogagents.datadoghq.com",
+		SilenceUsage: true,
+		RunE: func(c *cobra.Command, args []string) error {
+			if err := o.Complete(c, args); err != nil {
+				return err
+			}
+			if err := o.Validate(); err != nil {
+				return err
+			}
+
+			return o.Run()
+		},
+	}
+
+	cmd.Flags().StringArrayVar(&o.storedVersions, "storedVersions", []string{}, "used to define the status.storedVersion field in the CRD")
+
+	o.ConfigFlags.AddFlags(cmd.Flags())
+
+	return cmd
+}
+
+// Complete sets all information required for processing the command.
+func (o *Options) Complete(cmd *cobra.Command, args []string) error {
+	o.args = args
+	if len(args) > 0 {
+		o.crdName = args[0]
+	}
+
```

Comment:
should we also set here `o.storedVersion = args[1]` ?

Reply:
no I use a flags to get the storedVersions